### PR TITLE
Feature/release images

### PR DESCRIPTION
**Pull request by Aboisier**
This PR adds workflows for publishing missing images to GHCR. It also fixes Dockerfiles and the Devspace configuration so we now use GHCR images.

We are currently unable to publish images with certificates baked in. This will be addressed in a subsequent PR, as certificates will be moved to K8s secrets.

#### To-do
- [x] Add workflow for Python Job Base
- [x] Add workflow for Nuclei Job Base
- [x] Remove certs from Dockerfiles (I have a working PoC, just need to apply it to all services)

### CODE REVIEW

File: packages/backend/cron/service/certificates.yml
Comment (Aboisier) [inline packages/backend/cron/service/certificates.yml:1]: this file ultimately won't be committed.

File: devspace.yaml [hunk 1]
```diff
@@ -263,7 +272,13 @@ deployments:
               - name: STALKER_VERSION
                 value: ${STALKER_VERSION}
 
-            image: jobs-manager
+            volumeMounts:
+              - containerPath: /certs
+                volume:
+                  name: certs
+                  readOnly: false
```
Comment (lm-sec) [inline devspace.yaml:279]: pourquoi readonly? Mon impression c'est que c'est pas necessaire d'ecrire dedans?
```suggestion
                  readOnly: true
```
Ce meme commentaire va etre la une couple de fois

**Review by lm-sec — COMMENTED** [review]
Mon premier jet, je vais maintenant le deployer, voir ce que ca donne, tester une couple de trucs

File: devspace.yaml [hunk 2]
```diff
@@ -309,6 +329,12 @@ deployments:
               - name: STALKER_VERSION
                 value: ${STALKER_VERSION}
 
+            volumeMounts:
+              - containerPath: /certs
+                volume:
+                  name: certs
+                  readOnly: false
```
[the remark this answers is not in the document]
Comment (Aboisier) [reply devspace.yaml:336]: Oops, indeed, j'avais laissé plus open que pas assez le temps de tester 👍

**Review by lm-sec — APPROVED** [review]

**Pull request merged**